inspectit-oce-core/src/test/java/rocks/inspectit/oce/core/instrumentation/config/InstrumentationConfigurationResolverUnitTest.java

```diff
@@ -27,7 +27,7 @@
 import static org.mockito.Mockito.when;
 
 @ExtendWith(MockitoExtension.class)
-class InstrumentationConfigurationResolverTest {
+class InstrumentationConfigurationResolverUnitTest {
```

Comment:
Please keep unit tests named just `<ClassName>Test` and name all tests which are not unit tests differently, e.g. as we already did with `<ClassName>IntTest`